### PR TITLE
{numlib}[system/system] cuDNN v8.4.1.50 w/ CUDA 11.7.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
himem06.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/a3f9508f24bd25f0bfaac247fcb3589d for a full test report.

File: easybuild/easyconfigs/c/cuDNN/cuDNN-8.4.1.50-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is
+a GPU-accelerated library of primitives for deep neural networks."""
+
+toolchain = SYSTEM
+
+# note: cuDNN 8.4.1 is not specific to CUDA 11.6,
+# see also https://docs.nvidia.com/deeplearning/cudnn/support-matrix/index.html#cudnn-cuda-hardware-versions
+source_urls = ['https://developer.download.nvidia.com/compute/redist/cudnn/v%(version)s/local_installers/11.6/']
+local_cudnn_cuda_version = '%(version)s_cuda11.6'
```
Comment (jfgrimm) [inline easybuild/easyconfigs/c/cuDNN/cuDNN-8.4.1.50-CUDA-11.7.0.eb:14]: this is unused?

File: easybuild/easyconfigs/c/cuDNN/cuDNN-8.4.1.50-CUDA-11.7.0.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is
+a GPU-accelerated library of primitives for deep neural networks."""
+
+toolchain = SYSTEM
+
+# note: cuDNN 8.4.1 is not specific to CUDA 11.6,
+# see also https://docs.nvidia.com/deeplearning/cudnn/support-matrix/index.html#cudnn-cuda-hardware-versions
+source_urls = ['https://developer.download.nvidia.com/compute/redist/cudnn/v%(version)s/local_installers/11.6/']
```
Comment (jfgrimm) [inline easybuild/easyconfigs/c/cuDNN/cuDNN-8.4.1.50-CUDA-11.7.0.eb:13]: can't use the `version` template here, or else the download fails (it's `/v8.4.1/` not `/v8.4.1.50/`

**Review by jfgrimm — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/cuDNN/cuDNN-8.4.1.50-CUDA-11.7.0.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/c/cuDNN/cuDNN-8.4.1.50-CUDA-11.7.0.eb]: fixed in 3c81c78f4b

File: easybuild/easyconfigs/c/cuDNN/cuDNN-8.4.1.50-CUDA-11.7.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/c/cuDNN/cuDNN-8.4.1.50-CUDA-11.7.0.eb]: fixed in 3c81c78f4b

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
bear-pg0305u15a.bear.cluster - Linux RHEL 8.5, POWER, 8335-GTX (power9le), 1 x NVIDIA Tesla V100-SXM2-16GB, 470.57.02, Python 3.6.8
See https://gist.github.com/015fdee7215ea98bbae5b902b9cb4c92 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
bear-pg0103u04a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6330 CPU @ 2.00GHz (icelake), 1 x NVIDIA NVIDIA A30, 470.57.02, Python 3.6.8
See https://gist.github.com/a06778ab847b055272518d338a52c9d5 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**